test/extern/ferguson/c-array/c_array-anymanaged.good

```diff
@@ -0,0 +1 @@
+c_array-anymanaged.chpl:5: error: c_array element type cannot be generic
```

Comment:
I thought we had specialized messages for generic management type elsewhere.  It'd be good to do that here, if reasonable

Reply:
Those errors come from the compiler. Since this error originates in module code I think I would need to thread a new primitive through that checks if a class is anymanaged. Which I am not opposed to

That said, maybe `isClassType && (!isBorrowedType && !isOwnedType && !isSharedType && !isUnmanagedType)` is sufficient. I can try it

Reply:
That works! see https://github.com/chapel-lang/chapel/pull/27736/commits/5c7e02cfa4efc2b744b1eaabb6b2ed35cf59508a